sale_probability_amount/models/sale_order.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 Akretion (https://www.akretion.com).
+# @author Matthieu SAISON <matthieu.saison@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    probability = fields.Integer(
+        "Winning sale %",
+        default=lambda self: self.env.ref(
+            "sale_probability_amount.quotation_default_probability",
+            raise_if_not_found=False,
+        ).value,
```

Comment:
This is throwing an error in runboat build.  I suppose we should have a get-default method to handle the situation where the record has yet to exist?  We can't do `.value` anyway as it breaks the system if you delete the parameter record.